npm/css-src/sass/_drag-and-drop.scss

```diff
@@ -73,3 +72,9 @@
 .success-message-flexbox-item {
   margin-left: auto;
 }
+
+/* stylelint-disable */
```

Comment:
Could you add a class into the <p> tags so you can reference that and then you won't need the stylelint disable?